canonical-web-and-design/www.ubuntu.com
# Add redirects for /fips, /cc and /cis

## Done

Add redirects for  [www.ubuntu.com/cc](www.ubuntu.com/cc), [www.ubuntu.com/cis](www.ubuntu.com/cis) and [www.ubuntu.com/fips](www.ubuntu.com/fips) to  [this page](https://www.ubuntu.com/security/certifications)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Make sure [www.ubuntu.com/cc](www.ubuntu.com/cc), [www.ubuntu.com/cis](www.ubuntu.com/cis) and [www.ubuntu.com/fips](www.ubuntu.com/fips) takes you to [this page](https://www.ubuntu.com/security/certifications).


## Issue / Card

Fixes [#4982](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/4982)

It is related to [CanonicalLtd/ubuntu-advantage-client#420](https://github.com/CanonicalLtd/ubuntu-advantage-client/issues/420)
